presto-hive-metastore/src/test/java/com/facebook/presto/hive/metastore/thrift/InMemoryHiveMetastore.java

```diff
@@ -184,7 +183,7 @@ public synchronized MetastoreOperationResult createTable(MetastoreContext metast
             checkArgument(table.getSd().getLocation() == null, "Storage location for view must be null");
         }
         else {
-            File directory = new File(new Path(table.getSd().getLocation()).toUri());
+            File directory = new File(URI.create(table.getSd().getLocation()));
```

Comment:
Given changes in OSS won't break OSS build but can break Meta builds, can we introduce some tests in OSS so it will break as well when this implementation changes? This will help prevent future Meta build failures due to the same code changes.

Reply:
Possibly. I'm not yet sure if I should pursue this or not.

Reply:
Have we decided if we are going forward with this or not? If not, can we close this?